### PR TITLE
feat: allow overriding system script cell capacity

### PR DESCRIPTION
**Pull request by doitian**
This make the system script cell capacity determined.

### CODE REVIEW

File: spec/src/lib.rs [hunk 1]
```diff
@@ -617,10 +619,37 @@ impl SystemCell {
...
+                )
+                .into());
+            }
+            cell
+        } else {
+            builder.build_exact_capacity(Capacity::bytes(data.len())?)?
+        };
+
+        let data_len = Capacity::bytes(data.len())?;
+        dbg!(output_index);
```
Comment (xxuejie) [inline spec/src/lib.rs]: Should we remove those `dbg!` lines?

Comment (doitian) [reply spec/src/lib.rs]: sure

**Comment by zhangsoledad** [comment]
bors r=xxuejie,zhangsoledad

**Comment by zhangsoledad** [comment]
bors r=xxuejie,zhangsoledad,quake

**Pull request merged**